README.md

```diff
@@ -17,170 +17,11 @@
   - under the License.
   -->
 
-# Apache Iceberg Documentation Site
+# Apache Iceberg Documentation Site (Archived)
 
-This repository contains the documentation for [Apache Iceberg](https://github.com/apache/iceberg).
-It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.org.
+> [!WARNING]  
+> This repository is archived the website for Apache Iceberg documentation. This was moved to the [main Iceberg repository](https://github.com/apache/iceberg/tree/main/site).
```

Comment:
Minor phrasing: 
> This repository is archived the website for Apache Iceberg documentation

This repository an archive for the old Apache Iceberg documentation.